modules/react/menu/index.ts

```diff
@@ -2,7 +2,7 @@ export {useMenuModel} from './lib/useMenuModel';
 export {Menu} from './lib/Menu';
 
 export {useMenuCard} from './lib/MenuCard';
-export {useMenuItem} from './lib/MenuItem';
+export {useMenuItem, StyledMenuItem} from './lib/MenuItem';
```

Comment:
does this have to go into the upgrade guide? I know combobox in labs did this

Reply:
Possibly? It is a new feature...